manifests/oci-cloud-controller-manager.yaml

```diff
@@ -36,11 +36,13 @@ spec:
       serviceAccountName: cloud-controller-manager
       containers:
         - name: oci-cloud-controller-manager
-          image: wcr.io/oracle/oci-cloud-controller-manager:0.4.0
+          image: iad.ocir.io/oracle/oci-cloud-controller-manager:0.5.0
```

Comment:
We need to do a release ahead of time if we go down this route as 0.5.0 doesn't yet exist.